tests/test_iotools/test_atomic_models.py

```diff
@@ -62,10 +62,11 @@ def test_write_gemmi_model_pdb(self):
 
     def test_write_gemmi_model_cif(self):
         """Test test_write_gemmi_model_cif."""
-        path_input = "2dhb.cif"
-        os.system("wget https://files.rcsb.org/download/" + path_input)
-        model = read_gemmi_model(path_input)
-        path_output = "test_" + path_input
-        write_gemmi_model(path_output, model)
-        model = read_gemmi_model(path_output)
-        assert model.__class__ is gemmi.Model
+        # path_input = "2dhb.cif"
```

Comment:
Usually we would not leave left-over comments, but I'll merge anyway so that @fredericpoitevin can have a look.

Thanks!

Reply:
Yes officer Nina ;)

Hopefully Fred can quickly fix this with his gemmi skills and the comments will be gone in no time.